includes/controller/user_angeltypes_controller.php

```diff
@@ -38,19 +38,18 @@ function user_angeltypes_unconfirmed_hint()
         $unconfirmed_links[] = '<a href="'
             . url('/angeltypes', ['action' => 'view', 'angeltype_id' => $user_angeltype->angel_type_id])
             . '">' . htmlspecialchars($user_angeltype->angelType->name)
-            . ' (+' . $user_angeltype->count . ')'
+            . ' (+' . $user_angeltype->users_count . ')'
             . '</a>';
     }
 
     $count = $unconfirmed_user_angeltypes->count();
     return
         _e(
-            'There is %d unconfirmed angeltype.',
-            'There are %d unconfirmed angeltypes.',
+            'There is %d unconfirmed angel type. Angel type that needs approval:',
+            'There are %d unconfirmed angel types. Angel types that need approvals:',
```

Comment:
Technically, users are confirmed, not the angel type

Reply:
true but this is the original code. I'll fix it anyway.